server/setup_flow.go

```diff
@@ -311,6 +325,67 @@ func (p *Plugin) stepCloudUploadApp() flow.Step {
 		})
 }
 
+func (p *Plugin) stepCloudOAuthConfigure() flow.Step {
+	return flow.NewStep(stepCloudOAuthConfigure).
+		WithPretext("##### :white_check_mark: Step 2(a): Register an OAuth 2.0 Application in Jira").
+		WithText(fmt.Sprintf("Complete the following steps, then come back here to select **Configure**.\n\n"+
+			"1. Follow [these instructions](https://developer.atlassian.com/cloud/confluence/oauth-2-3lo-apps/#enabling-oauth-2-0--3lo-) to register an OAuth 2.0 application in Jira.\n"+
+			"2. Set the following values:\n"+
+			"	- Name: `Mattermost Jira Plugin - <your company name>`\n"+
+			"3. Select **Permissions** in the left menu. Next to the JIRA API, select **Add**\n"+
+			"4. Then select **Configure** and ensure following scopes are selected:\n"+
+			"   - Scopes: `%s`\n"+
+			"3. Copy the **Client ID** and **Secret** from the registered 0Auth Application's **Settings** page and keep it handy.\n", JiraScopes)).
```

Comment:
```suggestion
			"5. Copy the **Client ID** and **Secret** from the registered 0Auth Application's **Settings** page and keep it handy.\n", JiraScopes)).
```

You can also put `1.` on each of them, and the markdown renderer will automatically number them:

```
1. Item
1. Item
1. Item
```

1. Item
1. Item
1. Item